src/main/java/taskalyn/Database.java

```diff
@@ -0,0 +1,58 @@
+package taskalyn;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.IOException;
+import java.nio.file.NoSuchFileException;
+import java.util.List;
+
+public class Database {
+    private static final String DIRECTORY = "./data";
+    private static final String FILE_NAME = "taskalyn.txt";
+    private static final Path FILE_PATH = Paths.get(DIRECTORY, FILE_NAME);
+
+    public Database() {
+        try {
+            List<String> txtLines = Files.readAllLines(FILE_PATH);
```

Comment:
Perhaps you can rename this variable as `textLines` to avoid unnecessary abbreviations

Reply:
You're right! Thanks.